Doxyfile

```diff
@@ -1188,9 +1309,22 @@ HTML_EXTRA_STYLESHEET  = $(DRONE_BUILD_DIR)/code_docs/darxygen/custom.css \
 
 HTML_EXTRA_FILES       =
 
+# The HTML_COLORSTYLE tag can be used to specify if the generated HTML output
+# should be rendered with a dark or light theme.
+# Possible values are: LIGHT always generate light mode output, DARK always
+# generate dark mode output, AUTO_LIGHT automatically set the mode according to
+# the user preference, use light mode if no preference is set (the default),
+# AUTO_DARK automatically set the mode according to the user preference, use
+# dark mode if no preference is set and TOGGLE allow to user to switch between
+# light and dark mode via a button.
+# The default value is: AUTO_LIGHT.
+# This tag requires that the tag GENERATE_HTML is set to YES.
+
+HTML_COLORSTYLE        = AUTO_LIGHT
```

Comment:
Does this affect the generated document theme in any way?

Reply:
Yes. see https://www.doxygen.nl/manual/config.html#cfg_html_colorstyle, introduced by automatic update of doxygen file

Reply:
hmm, I'll have to take a look at that